default.nix

```diff
@@ -48,6 +48,8 @@ stdenv.mkDerivation rec {
         rasterio
 	    ipython
 	    nbformat
+	    ipyleaflet
+	    ipywidgets
```

Comment:
This is changed to the correct library on main (ipyleaflet --> mplleaflet)-- ipywidgets is fine. Not sure how to get this to update/match main default.nix; perhaps a rebase prior to merge?